scipy/fftpack/basic.py

```diff
@@ -215,8 +215,6 @@ def fft(x, n=None, axis=-1, overwrite_x=False):
 
             y(j) = sum[k=0..n-1] x[k] * exp(-sqrt(-1)*j*k* 2*pi/n), j = 0..n-1
 
-        Note that ``y(-j) = y(n-j).conjugate()``.
```

Comment:
The statements are wrong mathematically, so there's nothing to clarify. I put the explanation in the commit comments, not sure if that's visible to you. Here is a copy:

 Edited doc strings for fft functions

1) Some version of this appears repeatedly in help(fft), help(rfft), help(fftn):
Note that ``y(-j) == y(n-j).conjugate()``
This is wrong in two ways:
a) y(-j) is actually defined as y(n-j). 
b) perhaps the author meant to write ``y(j) == y(n-j).conjugate()``. But this is true only if the input x is a real-valued function, yet the statement appears even under complex fft. Better to omit this.
2) Under help(fft) we get this:
For `n` even, ``A[n/2]`` contains the sum of the positive and negative-frequency terms.  For `n` even and `x` real, ``A[n/2]`` will always be real.
This is misleading: There is only one frequency term at n/2, not a positive and a negative frequency. The statement gives the impression that n/2 is somehow handled differently from the other frequencies. Better to omit this.